### DISCUSSION TITLE
Issues with language selection when publishing an app

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#94] [Developer Feedback]
When publishing an application in the store, you need to specify the languages supported by the program. I did not find the Ukrainian language in the list of languages in the menu, although on the watch you can select this language in the menu. I also noticed that Urdu appears twice in the list of languages, perhaps this is a typo.

**Comment by MyoungXUE** [comment]
Hello, I have sent the problem to the relevant colleagues, please wait patiently.

Regarding the two occurrences you mentioned, can you provide a screenshot so that we can locate the problem?

**Comment by SashaCX75** [comment]
Here is a screenshot with two identical languages in the dropdown
![2023-03-02_18-34-37](https://user-images.githubusercontent.com/12464527/222493321-c9fe4e98-e4f1-4627-b576-19de23112f95.png)

**Reply by MyoungXUE** [reply]
Thank you very much for the important clue! We will solve this problem as soon as possible.
